content/en/docs/instrumentation/net/exporters.md

```diff
@@ -206,7 +211,7 @@ Next, install the Prometheus exporter:
 
 ```
 dotnet add package OpenTelemetry.Exporter.Prometheus
-dotnet add package OpenTelemetry.Extensions.Hosting --prerelease
+dotnet add package OpenTelemetry.Extensions.Hosting
```

Comment:
the line above needs fix as Prometheus exporter packages were split. Maybe a diff. PR.

Reply:
will add that back in as it sounds like the code won't work. I'll give it a try though